frontend/src/components/sprintboard/board.jsx

```diff
@@ -1,46 +1,51 @@
-import {mockSprints} from '../../../tests/utils/fake.js';
+import _ from 'lodash';
 import React from 'react/addons';
-// import Reflux from 'reflux';
-// import Actions from '../../actions/actions';
+import Reflux from 'reflux';
+import {SprintActions} from '../../actions/actions';
+import SprintStore from '../../stores/sprintStore';
 
-import SprintColumn from './column.jsx';
+import { DragDropContext } from 'react-dnd';
+import HTML5Backend from 'react-dnd/modules/backends/HTML5';
 
-let sprint = mockSprints(1, 3)[0];
+import SprintColumn from './column.jsx';
 
 let SprintBoard = React.createClass({
-  // mixins: [
-  //   Reflux.listenTo(HelloStore, 'onStoreUpdate')
-  // ],
+  mixins: [
+    Reflux.ListenerMixin
+  ],
 
   getInitialState() {
     return {
-      sprint: sprint,
-      sprintColumns: ['Backlog', 'Ready', 'In Progress', 'Done']
+      sprint: {},
+      sprintColumns: ['To Do', 'In Progress', 'Review', 'Done']
     };
   },
 
-  // onStoreUpdate(addressee) {
-  //   this.setState({
-  //     addressee: addressee
-  //   });
-  // },
+  componentDidMount() {
+    this.listenTo(SprintStore, this.onStoreUpdate);
```

Comment:
I don't know whether it is common practice nor do I remember where I borrowed this name, but it feels suitable. If I am not mistaken, the relationship between a component and a store is pretty dumb: store fires only one kind of event, and component listens to it. So basically there is only one function in the component that fires when the store triggers. And the store triggers when it updates.